packages/fhir-converter/src/lib/cda/cda.js

```diff
@@ -34,8 +34,7 @@ const valueTime00010101Regex = new RegExp('value="00010101000000*"s*/>', "g");
 const valueTime00010101Replacement = 'nullFlavor="NI" />';
 
 // TODO https://github.com/metriport/metriport-internal/issues/2004
-const singleAmpersandRegexOneOff = new RegExp('A&O x 3', "g");
-const singleAmpersandReplacementOneOff = 'A&amp;O x 3';
+const ampersandRegex = new RegExp('&(?!(?:#\\d+|#x[\\da-fA-F]+|amp|lt|gt|quot|apos);)', 'g');
```

Comment:
match & not followed by valid suffix values for &. See PR description

Reply:
Love the description! 👏👏👏

re: testing
- did we run against the large XML/test dataset?
- can we add unit tests for this, please? Like
   - examples that should be removed
   - examples that should not

Reply:
done and done. 
I also noticed that fhir-converter tests werent being run during CICD. I fixed that too

Reply:
I also noticed that the fhir-converter tests were never running during CICD on deployments. I updated that too. They arent run because they arent in the root package.jsons workspace.